content/responses/folder.yml

```diff
@@ -61,6 +61,54 @@ allOf:
           Specifies if this folder is owned by a user outside of the
           authenticated enterprise.
 
+      metadata:
+        x-box-field-variant: 3
+        type: object
+        description: |-
+          A list of key-value pairs of metadata objects within their
+          scopes.
+
+          To access the metadata for a file, first use the
```

Comment:
This says file. Do we maybe want to move the metadata into an attribute file and then reference it so we can re-use it across files and folders? Look for other uses of `$ref` in the codebase, like here.

https://github.com/box/box-openapi/blob/default/content/responses/file.yml#L193

Reply:
done